[program:gunicorn]
command=/Library/Frameworks/Python.framework/Versions/3.8/bin/gunicorn -c ./gunicorn.py main.wsgi:application
directory=/django_server所在的目录/django_server/server
numprocs=1           ; number of processes copies to start (def 1)
autostart = true     ; 在 supervisord 启动的时候也自动启动
startsecs = 1        ; 启动 1 秒后没有异常退出，就当作已经正常启动了
autorestart = true   ; 程序异常退出后自动重启
startretries = 3     ; 启动失败自动重试次数，默认是 3
user = root          ; 用哪个用户启动
stopsignal=INT               ; signal used to kill process (default TERM)
stopasgroup=true             ; send stop signal to the UNIX process group (default false)
killasgroup=true             ; SIGKILL the UNIX process group (def false)
;redirect_stderr = true          ; 把 stderr 重定向到 stdout，默认 false
;stdout_logfile_maxbytes = 20MB  ; stdout 日志文件大小，默认 50MB
;stdout_logfile_backups = 10     ; stdout 日志文件备份数
;stdout_logfile=/django_server所在的目录/django_server/logs/gunicorn.log       ; log 日志  gunciorn已经打印了日志 这里就不设置了 这里设置输出日志反而没有
;stderr_logfile=/django_server所在的目录/django_server/logs/gunicorn.error     ; 错误日志的绝对路径
